packages/components/src/components/password-meter/password-meter.jsx

```diff
@@ -12,13 +12,16 @@ const PasswordMeter = ({ children, has_error, input, custom_feedback_messages })
     const [feedback, setFeedback] = React.useState();
 
     React.useEffect(() => {
+        let is_subscribed = true;
```

Comment:
why call it `is_subscribed`? 🤔

Reply:
Suggestions are welcome :smile: 
Perhaps `is_state_update_allowed` or `is_async_op_allowed`? :thinking:

Reply:
you can utilize `useIsMounted` hook here